app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module V1
+    class TargetsController < Api::V1::ApiController
+      def index
+        @targets = Target.all
+        render json: @targets
+      end
+
+      def create
+        @target = current_user.targets.create!(target_params)
+        if @target.save
+          render json: { success: true, target: @target }
+        else
+          render json: { success: false }
+        end
+      end
+
+      def destroy
+        if @target.find(params[:id]).destroy!
```

Comment:
you can move the `.find` method to a separated one, just to get the `Target`, probably you will reuse it when code the `show` endpoint.
So, you can do something like: 
```
...
def destroy 
    if target.destroy! # here you are calling the method below
end 
...
private 

def target
    @target ||= Target.find(params[:id])
end 
...
```
You can see that I use `||=` for the assignment of the `@target`, this is the same as the `=` but if the `@target` has already a value, it avoid the DB call.. si it's more performant.